hack/local-up-cluster.sh

```diff
@@ -823,12 +823,17 @@ EOF
       --v=${LOG_LEVEL} 2>&1 &
     PROXY_PID=$!
 
+    cat <<EOF > /tmp/kube-scheduler.yaml
```

Comment:
Could we remove this file after stopping? I found that `/tmp/kube-proxy.yaml` is also doesn't removed but I don't think that it's a good pattern that we should follow.

Reply:
@php-coder Update in the cleanup func

Reply:
I realize this is just copying the pattern used by `/tmp/kube-proxy.yaml`, but this should probably be using `mktemp` instead of hard-coding a static filename.